en/tutorial-base.md

```diff
@@ -72,26 +72,34 @@ tutorial/
 
-Before continuing, please be sure you've successfully [installed Phalcon](/[[language]]/[[version]]/installation) and have setup either [nginX](/[[language]]/[[version]]/setup#nginx), [Apache](/[[language]]/[[version]]/setup#apache) or [Cherokee](/[[language]]/[[version]]/setup#cherokee).
+Otherwise if you want to use Nginx here are some additional setup [here](/[[language]]/[[version]]/setup#nginx)
+
+Apache can also be used with these additional setup [here](/[[language]]/[[version]]/setup#apache)
+
+Finally, if you flavor is Cherokee use the setup [here](/[[language]]/[[version]]/setup#cherokee)
 
 <a name='bootstrap'></a>
 ## Bootstrap
-The first file you need to create is the bootstrap file. This file is very important; since it serves as the base of your application, giving you control of all aspects of it. In this file you can implement initialization of components as well as application behavior.
+The first file you need to create is the bootstrap file. This file acts as the entry-point and configuration for you application. In this file you can implement initialization of components as well as application behavior.
 
-Ultimately, it is responsible for doing 3 things:
-
-* Setting up the autoloader.
-* Configuring the Dependency Injector.
-* Handling the application request.
+This file handles 3 things:
+- Registration of component autoloaders.
+- Configuring **Services** and registering them with the **Dependency Injection** context.
+- Resolving the application's HTTP requests.
 
 <a name='autoloaders'></a>
 ### Autoloaders
-The first part that we find in the bootstrap is registering an autoloader. This will be used to load classes as controllers and models in the application. For example we may register one or more directories of controllers increasing the flexibility of the application. In our example we have used the component `Phalcon\Loader`.
+Autoloaders leverage a [PSR-4](http://www.php-fig.org/psr/psr-4/) compliant file loader running through the Phalcon C extension. Common things that should be added to the Autoloader are your **Controllers** and **Models**. You can register **directories** which will search for files within the application's namespace. (If you want to read about other ways that you can use Autoloaders head [here](https://olddocs.phalconphp.com/en/3.0.0/reference/loader.html#class-autoloader))
 
-With it, we can load classes using various strategies but for this example we have chosen to locate classes based on predefined directories:
+To start lets register our apps controllers and models directories.
+Don't forget to include the loader from `Phalcon\Loader`.
```

Comment:
I was under the impression that the new line would cover this but I will double check.